src/pages/index.js

```diff
@@ -70,19 +138,77 @@ function handleImageClick(cardData) {
   previewImageModal.open(cardData);
 }
 
+let cardToDelete;
+
+function handleDeleteCardClick(card) {
+  console.log("Card to delete:", card);
+  deleteModal.open();
+  cardToDelete = card;
+  confirmDeleteBtn.addEventListener("click", handleConfirmDelete);
```

Comment:
You need to set the click listener inside `PopupDelete` because any event listeners should be set in `setEventListeners`

Reply:
Thank you for all the feedback, but I'm confused about this error. Could you clarify or show an example of how I'd put it in the PopupDelete class?

Reply:
> @gennady-bars Thank you for all the feedback, but I'm confused about this error. Could you clarify or show an example of how I'd put it in the PopupDelete class?

You've done everything right